packages/launchpad-navigator/src/floating-navigator/index.tsx

```diff
@@ -25,6 +25,12 @@ const FloatingNavigator = ( { siteSlug, toggleLaunchpadIsVisible }: FloatingNavi
 
 	const setLaunchpadIsVisible = toggleLaunchpadIsVisible || ( () => {} );
 
+	const onTaskClick: EventHandlers[ 'onTaskClick' ] = ( task: Task ) => {
+		if ( task.calypso_path !== undefined ) {
```

Comment:
I think we should rely on `task.useCalypsoPath` as well as `task.calypso_path`. That's the final decision on which path to use in our default handling.
```suggestion
		if ( task.useCalypsoPath && task.calypso_path !== undefined ) {
```
https://github.com/Automattic/wp-calypso/blob/ed8fa4039de667a71ac675bce4b180c79422e659/packages/launchpad/src/setup-actions.ts#L162